pandas/tests/scalar/test_timedelta.py

```diff
@@ -953,6 +770,162 @@ def test_isoformat(self):
         expected = 'P0DT0H1M0S'
         assert result == expected
 
+
+class TestTimedeltaConstructor(object):
```

Comment:
rather than just moving things around, can you split into sub-modules
``scalar/timedelta/test_construction`` (and so on), ok to just move these (and just move the main file to scalar/timedelta/test_timedelta for now as well